client/app/telecom/telephony/line/calls/forward/telecom-telephony-line-calls-forward.controller.js

```diff
@@ -239,9 +257,6 @@ angular.module('managerApp').controller('TelecomTelephonyLineCallsForwardCtrl',
     self.saved = angular.copy(self.options);
     tucTelecomVoip.fetchAll().then((billingAccounts) => {
       self.listBillingAccounts = billingAccounts;
-      self.listBillingAccounts.unshift({ billingAccount: null, description: 'Tous les groupes' });
```

Comment:
`Tous les groupes` option must be kept from the `<select>` element.

Reply:
Could you please confirm that `billingAccounts` are correctly sorted by description? It seems not be the case.

Thanks!

Reply:
about sorting: yes, see code ` | orderBy:'description'` and attached screenshot (had to resize select as I can't screenshot an opened select)
For the "tous les groupes", I removed it, either it will be sorted along with the other descriptions. Removing it produces a blank choice, which I think is fine as it means "no billing account selected for filter"
![Screenshot](https://user-images.githubusercontent.com/995738/62056481-09853a80-b21e-11e9-8da5-add5dd696888.png)

Reply:
I think I understood: billing accounts as groups (HTML optgroup) in the select lists on the left (targets of the forward) are not sorted, it's right. I didn't had time to do this when I first wanted to fix the mail issue. This may be another improvement.

Reply:
Did a new commit on my branch which sorts lists by group then service description ;)

Reply:
and another one which restores "all billing accounts" select option and makes it translation ready

Reply:
OK. Got it!

Thanks @julieni for your feedback.